KMM_Showcase/src/main/res/layout/fragment_job_details.xml

```diff
@@ -2,7 +2,49 @@
 <androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
     android:layout_width="match_parent"
-    android:layout_height="match_parent">
+    android:layout_height="match_parent"
+    android:paddingTop="@dimen/margin_default">
+
+    <TextView
+        android:id="@+id/tvJotDetailPriceByAreaTitle"
```

Comment:
Oops I think it should be `Job` 😂 

```suggestion
        android:id="@+id/tvJobDetailPriceByAreaTitle"
```

same for all ids below

Reply:
Oh wrong copy-paste, let me update. 😂